spyder_kernels/customize/spydercustomize.py

```diff
@@ -674,6 +677,49 @@ def runfile(filename, args=None, wdir=None, namespace=None, post_mortem=False):
 builtins.runfile = runfile
 
 
+def runcell(cell_name, filename):
+    """
+    Runs a code cell from an editor as a file.
+
+    Currently looks for code in in a ipython property called `cell_code`.
+    This property must be set by the editor prior to calling this function
+    This function deletes the contents of `cell_code` upon completion.
+
+    The filename is needed to allow for proper traceback links.
+
+    The cell_name is used currently as a reference in the history log of which
+    cell was run with the command. It is currently unused in the function.
+
+    Parameters
+    ----------
+
+    cell_name: The name of the code cell (string)
```

Comment:
What docstring convention is this? It doesn't like like Googledoc, Numpydoc, plain reST or javadoc/epytext/etc. I don't believe we've ever formally decided on one since our docstrings are rather skimpy, but those we do have are either Googledoc or Numpydoc so probably best to stick with one of those, or at the very least a common format.

Reply:
Yeah, it's not anything.  I'm going to go for Numpydoc because I'm most familiar with that.